chiselFrontend/src/main/scala/chisel3/core/Aggregate.scala

```diff
@@ -556,6 +556,11 @@ class Bundle(implicit compileOptions: CompileOptions) extends Record {
     case _ => None
   }
 
+  // If this Data is an instance of an inner class, record enclosing class
+  // This is only used for cloneType, and is mutable to allow autoCloneType to preserve the
+  // original outerInstance.
+  private var _outerInst: Option[Object] = Builder.currentModule
```

Comment:
Comments might want to note that this is only a guess, because Builder.currentModule may be the wrong thing.